requirements/requirements-testing.txt

```diff
@@ -1,10 +1,16 @@
 -r requirements-general.txt
 
-pytest
-pytest-cov
-mypy
-black
-isort
-flake8
-pytest-flask
-debugpy
\ No newline at end of file
+# testing
+pytest==7.2.1
```

Comment:
Why do you fix the versions, especially for development helpers?

Reply:
Because it lead to errors in local environment vs. CI. The local environment is cached and is not rebuild when a new version of some package is released, while the CI does not use any cache (and even if it did, not the same one). In this specific case, black was updated and wanted to reformat some files in the CI, while I had to manually update my local environment after I debugged the error and found the source.

Reply:
Also, the backend also uses pinned dev tool versions